matchmake-extension/create_matchmake_session.go

```diff
@@ -7,12 +7,13 @@ import (
 	matchmake_extension "github.com/PretendoNetwork/nex-protocols-go/matchmake-extension"
 )
 
-func createMatchmakeSession(err error, client *nex.Client, callID uint32, anyGathering *nex.DataHolder, message string, participationCount uint16) uint32 {
+func createMatchmakeSession(err error, packet nex.PacketInterface, callID uint32, anyGathering *nex.DataHolder, message string, participationCount uint16) uint32 {
 	if err != nil {
 		common_globals.Logger.Error(err.Error())
 		return nex.Errors.Core.InvalidArgument
 	}
 
+	client := packet.Sender()
 	server := client.Server()
```

Comment:
While you are here, can you update these to get the server to the common protocol?

Reply:
You want this changed to this, yes?

```suggestion
	server := commonMatchmakeExtensionProtocol.server
```

Reply:
Correct!